diff --git a/Libs/gli-0.5.1.0/include/gli/core/save_dds.inl b/Libs/gli-0.5.1.0/include/gli/core/save_dds.inl
index 56ea078..d122e51 100644
--- a/Libs/gli-0.5.1.0/include/gli/core/save_dds.inl
+++ b/Libs/gli-0.5.1.0/include/gli/core/save_dds.inl
@@ -29,7 +29,7 @@
 namespace gli{
 namespace detail
 {
-	glm::uint32 getMaskRed(format const & Formatl)
+	inline glm::uint32 getMaskRed(format const & Formatl)
 	{
 		switch(Formatl)
 		{
@@ -43,7 +43,7 @@
 		}
 	}
 
-	glm::uint32 getMaskGreen(format const & Formatl)
+	inline glm::uint32 getMaskGreen(format const & Formatl)
 	{
 		switch(Formatl)
 		{
@@ -56,7 +56,7 @@
 		}
 	}
 
-	glm::uint32 getMaskBlue(format const & Formatl)
+	inline glm::uint32 getMaskBlue(format const & Formatl)
 	{
 		switch(Formatl)
 		{
@@ -68,7 +68,7 @@
 		}
 	}
 
-	glm::uint32 getMaskAlpha(format const & Formatl)
+	inline glm::uint32 getMaskAlpha(format const & Formatl)
 	{
 		switch(Formatl)
 		{
diff --git a/Libs/gli-0.5.1.0/include/gli/core/storage.hpp b/Libs/gli-0.5.1.0/include/gli/core/storage.hpp
index 5e7d2b6..e490318 100644
--- a/Libs/gli-0.5.1.0/include/gli/core/storage.hpp
+++ b/Libs/gli-0.5.1.0/include/gli/core/storage.hpp
@@ -38,7 +38,9 @@
 #include <cstring>
 
 // GLM
+#ifndef GLM_FORCE_RADIANS
 #define GLM_FORCE_RADIANS
+#endif
 #include <glm/glm.hpp>
 #include <glm/gtx/component_wise.hpp>
 #include <glm/gtx/gradient_paint.hpp>